h2/src/main/org/h2/engine/Session.java

```diff
@@ -627,6 +634,10 @@ public Command prepareLocal(String sql) {
         return command;
     }
 
+    void releaseDatabaseObjectId(int id) {
```

Comment:
this method has a verb form i.e. "do it now", perhaps a better name, something like "markDatabaseObjectIdForDelayedRelease"
or something else equally obvious